samples/Config.CustomConfigurationProviders.Sample/Config.CustomConfigurationProviders.Sample.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It's at actually not a web project. It only has web.config because it's showing how to use configuration to read xml.